lib/rpmsg/rpmsg_virtio.c

```diff
@@ -636,11 +639,20 @@ static int rpmsg_virtio_ns_callback(struct rpmsg_endpoint *ept, void *data,
 	if (ns_msg->flags & RPMSG_NS_DESTROY) {
 		if (_ept)
 			_ept->dest_addr = RPMSG_ADDR_ANY;
+		if (_ept && _ept->release_cb) {
```

Comment:
already tested in  `rpmsg_ept_incref`

Reply:
642 is to judge local variable ept_decref